### ISSUE TITLE
Use list items for accessibility #a11y

### ISSUE THREAD

**Issue by growdigital** [netlify/netlify-cms#1488]
**- Do you want to request a *feature* or report a *bug*?**

Bug

**- What is the current behavior?**

HTML elements in the app which ought to be lists are actually nested divs ie in header, aside and main.

This is a continuation of #1486 'Use HTML elements for accessibility'. I put it in a separate issue as it is considerably more complex.

**- If the current behavior is a bug, please provide the steps to reproduce.**

Using a screenreader, a list of items should be announced but they won't be, as it's nested divs. Alternatively, view source and look inside the header, sidebar and main areas.

**- What is the expected behavior?**

A screenreader announces a list of options, which helps navigation

**- Please mention your versions where applicable.**

Netlify CMS version: 2.0.0-dev-dev
Browser version: 61.0

Node.JS version: v8.9.1
Operating System: MacOS 10.13.5
